src/formElementPlugins/ApplicationViewWrapper.tsx

```diff
@@ -38,17 +38,17 @@ const ApplicationViewWrapper: React.FC<ApplicationViewWrapperProps> = (props) =>
     })
   }, [])
 
+  useEffect(() => {
```

Comment:
I think this is the problem with why the error status is not showing up. The forceValidation causes the field to get saved, but the actual "Value" its displaying is local to the plugin, and doesn't get changed. The `currentResponse.text` that you're passing in is NOT the current "value", as `currentResponse` doesn't get updated until onLoseFocus event in plugin.

Two options I can think of:

1. Pass `forceValidation` down into plugin, then do the useEffect in there to update "value" on `forceValidation`. 
2. Move `const [value, setValue] = useState<string>(initialValue?.text)` from each plugin up into the wrapper and just pass it down to the plugin. Then, in the Wrapper, we can update the actual "value" using "setValue" in the `forceValidation` useEffect hook.

Definitely think option 2 is better, as it moves common functionality out of the plugins and into the wrapper.

Reply:
Thanks for the suggestion! I'll try this out. : )